_site.yml

```diff
@@ -9,17 +9,17 @@ navbar:
     - text: "Scenarios"
       icon: fa-virus
       href: scenarios.html
```

Comment:
Is the idea that people click on "Scenarios" where the menu then unfolds below to get to the definitions? It doesn't seem to work for me and I don't think it's very obvious.

Perhaps we could have an "Overview" menu item at the top that links to this page?